app/helpers/admin/enterprises_helper.rb

```diff
@@ -14,6 +14,10 @@ def select_only_item(producers)
       producers.size == 1 ? producers.first.id : nil
     end
 
+    def can_connect_apps?(enterprise)
```

Comment:
What do you think about naming this for the condition? It's what the message talks about. Forgive me if I'm overthinking it now..
```
    def managed_by_user?(enterprise)
```

Reply:
Yes, that sounds good. It is clearer for now because you know what the method is doing. If the condition changes in the future then we may need a different name again. Anyway, I think either is good but I applied your recommendation.